wiki/Tournaments/otst!/2019/en.md

```diff
@@ -0,0 +1,224 @@
+---
+tags:
+- osu! TV Size Tournament 2019
+- otst! 2019
+- otst!2019
+---
+# osu! TV Size Tournament 2019
+
+![otst! 2019 Logo](img/logo.png)
+
+The **osu! TV Size Tournament 2019** (***otst! 2019***) is a double elimination four-on-four osu! tournament hosted by ![][flag_TH] [Trigonoculus](/users/7627013) and ![][flag_TH] [Seaweed](/users/5151214). osu! TV Size Tournament features variety of TV sized (usually 90 seconds) maps in its mappools. It is the third installment of the osu! TV Size Tournament.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-01-26/2019-02-10 |
+| Live Drawings | 2019-02-16 (14:00 UTC) |
+| Group Stage | 2019-02-23/2019-03-03 |
+| Round of 32 | 2019-03-09/2019-03-10 |
+| Round of 16 | 2019-03-16/2019-03-17 |
+| Quarterfinals | 2019-03-23/2019-03-24 |
+| Semifinals | 2019-03-30/2019-03-31 |
+| Finals | 2019-04-06/2019-04-07 |
+| Grand Finals | 2019-04-13 |
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu! TV Size Tournament 2019 winners. It is possible that these items change with every installment of the osu! TV Size Tournament.
```

Comment:
(applies to quite a few other tournament articles) not sure if this is even useful information, reads like fluff bc it's kind of obvious that the prizes aren't exactly the same as other tournaments in the series (since they are listed right here)

Reply:
yeah, lets remove it going forward.